deploy-service/common/src/main/java/com/pinterest/deployservice/bean/BuildBean.java

```diff
@@ -117,11 +118,11 @@ public void setArtifact_url(String artifact_url) {
     }
 
     public String getScm() {
-        return scm;
+       return StringUtils.capitalize(scm);
```

Comment:
Updated the getter and setter since I could not modify the @JsonProperty value directly